pyomo/neos/tests/test_neos.py

```diff
@@ -149,8 +149,11 @@ def test_minto(self):
     def test_mosek(self):
         self._run('mosek')
 
-    def test_octeract(self):
-        self._run('octeract')
+    # [16 Jul 24] Octeract is erroring.  We will disable the interface
+    # (and testing) until we have time to resolve #3321
+    #
+    # def test_octeract(self):
```

Comment:
I would instead add a `unittest.skip` but I'm fine with this.

Reply:
I thought about that - actually, I thought about making it an `xFail`, but I don't want to leave an interface in place that we know doesn't work.  There is use for the "noisy skip", but I ended up going with the open issue as the "right" place to leave the breadcrumb.

Reply:
Totally fair and agreeable.